openstack/02a-MAClow/IEEE802154E.h

```diff
@@ -255,7 +276,9 @@ typedef struct {
     PORT_TIMER_WIDTH          radioOnInit;             // when within the slot the radio turns on
     PORT_TIMER_WIDTH          radioOnTics;             // how many tics within the slot the radio is on
     bool                      radioOnThisSlot;         // to control if the radio has been turned on in a slot.
-
+    bool                      UseDefaultRFConfig;      // tells if we are in a state to use the default RF config (e.g. shared state) or use a custom RF config (e.g. an active negotiated slot).
+    uint8_t                   DefaultRFConfig;          // defines the RF config to use by default (such as on shared slots, and perhaps for retransmissions later?). This currenly represents only level of Tx power. Later it should be a struct representing heterogeneous configuration.

```

Comment:
I don't understand how this variable is related to slot type?
If this is for TxPower, just use variable name like txpower. 
However, tx power should not be part of the IEEE802.15.4